cardano-node/app/cardano-node.hs

```diff
@@ -155,13 +156,12 @@ parseNodeArgs =
 parseNodeAddress :: Parser NodeAddress
 parseNodeAddress = NodeAddress <$> parseHostAddr <*> parsePort
 
-parseHostAddr :: Parser (Maybe IP.IP)
```

Comment:
Do we really want to make the `--host-addr` option mandatory?

I think recent work by @karknu was about going in the opposite direction?

Reply:
--host-addr should not be mandatory, keeping it blank makes it possible to have a node who use both IPv4 and IPv6 addresses at the same time.
Users never had to specify the source address when running cardano-sl.

Reply:
@karknu  I have to re-implement this with `attoparsec` so I will undo the change.